docs/ml/vision/detection.md

```diff
@@ -247,7 +247,7 @@ If you intend to use the detector with a camera that is part of your robot, you
          "type": "detections",
          "attributes": {
            "confidence_threshold": 0.5,
-           "detector_name": "my_detector"
+           "detector_name": "my_vision_service"
```

Comment:
i believe this is right (it's the vision service), but should have <> for consistency.

Reply:
agreed, <camera-name> has brackets below, but this one has quotes

Reply:
Ah I see now, updated with brackets for `my_vision_service` then, you're both right here.